src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -75,10 +76,8 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
                 type: 'docker-coreclr',
                 request: 'launch',
                 preLaunchTask: 'build',
-                dockerBuild: {
-                },
-                dockerRun: {
-                }
+                dockerBuild: {},
+                dockerRun: {}
```

Comment:
If this is on autosave, look into what settings you have here. Changes like these should not be happening.

Reply:
This was intentionally changed; to match what we have in package.json.